pkg/lib/cockpit-components-modifications.css

```diff
@@ -0,0 +1,30 @@
+.modifications-table {
+    border: solid 2px #f1f1f1;
```

Comment:
I keep having issues with fuzzy exports in Inkscape, so it might have looked like 2px on the mockup.
I intended the style to be like dialog-list-ct, like in the storage dialogs
![Screenshot from 2019-05-10 18-42-34](https://user-images.githubusercontent.com/209895/57542981-6544d380-7353-11e9-98da-35d21f8ebe75.png)

Reply:
Here's the reason for fuzzy exports:

Inkscape switched from 90 DPI (which is weird, but it used it for _years_) to 96 DPI. Make sure you have the correct DPI in the export dialog.

Inkscape warns you when you open an older file and lets you fix it... but exporting settings might not be obvious.

If in doubt, you can always put int the pixel size you expect and it'll change the DPI accordingly.